docs/pull_request_template.md

```diff
@@ -1,16 +1,11 @@
-## Description
+# Description
 
-## Depends On
 
-If your pull request depends on functionality that is not yet merged to `master`,
-please note that here. ex: "Depends on #1"
+# TODO
 
-## TODO
+<!-- Use this section for draft pull requests -->
```

Comment:
I think it's better to indicate explicitly what items are done for all PRs (even non-draft: 
```suggestion
<!-- Use <del></del> for items that are not required for this PR -->
```